dev_requirements.txt

```diff
@@ -5,7 +5,7 @@ git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=
 
 # install latest changes in dbt-spark
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-spark.git#egg=dbt-spark
+git+https://github.com/dbt-labs/dbt-spark.git@d4c1498703b0c1c2a1a0e9913a46675f27a15b47#egg=dbt-spark
```

Comment:
This line should be reverted once https://github.com/dbt-labs/dbt-spark/pull/320 is merged.